test/integration/epacems_test.py

```diff
@@ -70,7 +70,7 @@ def test_epacems_subset_input_validation(epacems_year_and_state, epacems_parquet
 def test_epacems_parallel(pudl_settings_fixture, pudl_ds_kwargs, tmpdir_factory):
     """Test that we can run the EPA CEMS ETL in parallel."""
     epacems_settings = EpaCemsSettings(
-        years=[2019, 2020], states=["ID", "ME"], partition=True
+        years=[2019, 2020, 2021], states=["ID", "ME"], partition=True
```

Comment:
I left the 3rd year in there because 2021 is (at this point) still a different kind of year -- it's a year for which we only have eia860m data. It's only a very small amount of additional data that it has to download and cache locally.